src/test/java/lotto/domain/LottoResultTest.java

```diff
@@ -0,0 +1,25 @@
+package lotto.domain;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+public class LottoResultTest {
+    @Test
+    void create() {
+        Lotto winLotto = new Lotto(new String[]{"1", "2", "3", "4", "5", "6"});
```

Comment:
```suggestion
        Lotto winLotto = new Lotto("1,2,3,4,5,6");
        또는 
        Lotto winLotto = new Lotto(1, 2, 3, 4, 5, 6);
```
위 두 가지 방법으로 Lotto를 생성할 때 수 있다면 테스트 코드가 훨씬 더 깔끔해 지지 않을까?
Lotto에 생성자를 추가해 위와 같이 접근해 본다.